docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLogMeanVarianceFixZero.cs

```diff
@@ -0,0 +1,82 @@
+                new DataPoint(){ Features = new float[5] {-1,-1,-1, 1, 1} }
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
+            // Uses Cumulative distribution function as output.
+            var normalize = mlContext.Transforms.NormalizeLogMeanVariance(true, "Features", useCdf: true);
+
+            // NormalizeLogMeanVariance normalizes the data based on the computed mean and variance of the logarithm of the data.
+            var normalizeNoCdf = mlContext.Transforms.NormalizeLogMeanVariance(true, "Features", useCdf: false);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var normalizeTransform = normalize.Fit(data);
+            var transformedData = normalizeTransform.Transform(data);
+            var normalizeNoCdfTransform = normalizeNoCdf.Fit(data);
+            var noCdfData = normalizeNoCdfTransform.Transform(data);
+            var column = transformedData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in column)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // Expected output:
+            //  0.1587, 0.1587, 0.8654, 0.0000, 0.8413
+            //  0.8413, 0.8413, 0.5837, 0.0000, 0.0000
+            //  0.0000, 0.0000, 0.0940, 0.0000, 0.0000
+            //  0.0000, 0.0000, 0.0000, 0.0000, 0.1587
+
+            var columnFixZero = noCdfData.GetColumn<float[]>("Features").ToArray();
+            foreach (var row in columnFixZero)
+                Console.WriteLine(string.Join(", ", row.Select(x => x.ToString("f4"))));
+            // Expected output:
+            //  2.0403, 2.0403, 4.0001, 0.0000, 5423991000000000000000000000000000000.0000
+            //  4.0806, 4.0806, 2.6667, 0.0000,-5423991000000000000000000000000000000.0000
+            //  0.0000, 0.0000, 1.3334, 0.0000, 0.0000
+            // -2.0403,-2.0403,-1.3334, 0.0000, 0.0159
+
+            // Let's get transformation parameters. Since we work with only one column we need to pass 0 as parameter for GetNormalizerModelParameters.
+            // If we have multiple columns transformations we need to pass index of InputOutputColumnPair.
+            var transformParams = normalizeTransform.GetNormalizerModelParameters(0) as CdfNormalizerModelParameters<ImmutableArray<float>>;
+            Console.WriteLine("The 1-index value in resulting array would be produce by:");
+            Console.WriteLine($"y = 0.5* (1 + ERF((Math.Log(x)- {transformParams.Mean[1]}) / ({transformParams.StandardDeviation[1]} * sqrt(2)))");
```

Comment:
We have equation for `columnFixZero` now! May we have the same thing for `column`?

Reply:
I just copied the equation from the LogMeanVariance normalizer sample :) 
There is an equation for both settings of FixZero and for both Cdf and no Cdf. So I think it's complete. 

---
In reply to: [298258190](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298258190) [](ancestors = 298258190)